From 2a44c69dfe63dfdc53248484e5ac94022e942e62 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Fri, 23 Nov 2018 21:09:55 +0900
Subject: [PATCH 525/909] arm64: dts: r8a77995-draak: Set max dot clock for vga

commit 6c59e13ad0dbc11c40e0120104eb2c80ecea3b52 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

Add option to specify the maximum dot clock that can be
output on VGA connector in Draak board.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a77995-draak.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a77995-draak.dts b/arch/arm64/boot/dts/renesas/r8a77995-draak.dts
index 5d22f1206575..da02ead038a1 100644
--- a/arch/arm64/boot/dts/renesas/r8a77995-draak.dts
+++ b/arch/arm64/boot/dts/renesas/r8a77995-draak.dts
@@ -31,6 +31,7 @@
 
 		width = <800>;
 		height = <600>;
+		max-pixelclock = <50000>;
 
 		port {
 			vga_in: endpoint {
-- 
2.17.1

